src/Microsoft.SqlTools.ServiceLayer/DacFx/GenerateTSqlModelOperation.cs

```diff
@@ -37,16 +36,15 @@ public GenerateTSqlModelResult GenerateTSqlModel()
 
                 var model = new TSqlModel(version, options);
                 // read all sql files
-                if (Parameters.FilePaths?.Length > 0)
+                if (Parameters.FilePaths.Length > 0)
```

Comment:
You don't need this check, the foreach will just not do anything if there's no file paths